extensions.json

```diff
@@ -0,0 +1,117 @@
+{
+  "version": 0.1,
+  "extensions": [
+    {
+      "name": "CustomCSS",
+      "description": "A FreshRSS extension which give ability to create user-specific CSS rules to apply in addition of the actual theme",
+      "version": 0.1,
+      "author": "Marien Fressinaud",
+      "url": "https://github.com/FreshRSS/Extensions",
+      "type": "gh-subdirectory"
+    },
+    {
+      "name": "CustomJS",
+      "description": "A FreshRSS extension which give ability to create user-specific Javascript rules to apply in addition of the actual theme",
+      "version": 0.1,
+      "author": "₣rans de Jonge",
```

Comment:
Is it just my phone or did you use some odd glyph instead of F?

Reply:
I double checked that yesterday as well as it looked unfamiliar to me as well ... but it comes from https://github.com/FreshRSS/Extensions/blob/master/xExtension-CustomJS/metadata.json

Reply:
I… typod my own name with the French franc? O_o That's a new one.

I guess I must've accidentally used some Compose key sequence.

Reply:
I changed your name in this file, but the metada.json is still open